pkg/controllers/nodeclaim/lifecycle/liveness_test.go

```diff
@@ -78,6 +81,12 @@ var _ = Describe("Liveness", func() {
 			ExpectFinalizersRemoved(ctx, env.Client, nodeClaim)
```

Comment:
We should validate this: Do we have a test that we succeed when the NodeClaim has no owning NodePool?

Reply:
Added